src/nvim/ex_docmd.c

```diff
@@ -6300,15 +6300,18 @@ static void ex_stop(exarg_T *eap)
     if (!eap->forceit) {
       autowrite_all();
     }
+    apply_autocmds(EVENT_VIMSUSPEND, NULL, NULL, FALSE, NULL);
     ui_cursor_goto((int)Rows - 1, 0);
     ui_linefeed();
     ui_flush();
     ui_call_suspend();  // call machine specific function
+
     ui_flush();
     maketitle();
     resettitle();  // force updating the title
     redraw_later_clear();
     ui_refresh();  // may have resized window
+    apply_autocmds(EVENT_VIMRESUME, NULL, NULL, FALSE, NULL);
```

Comment:
linter is complaining about this, FALSE should be lowercase. same with line 6303.

run `make lint` locally, then you will find out more quickly :)

Reply:
yes, I saw that. there are other `FALSE` in the same file it's not complaining about them, so I thought it was fine :disappointed: 
Does the rule is for parameters, i guess so.

I'll make the lint changes. thanks a lot!

Reply:
the linter only checks lines (+/-1) that you touched, otherwise we would have too many lint issues.